pkg/api/v1/router_server_components.go

```diff
@@ -57,12 +58,7 @@ func (r *Router) serverComponentGet(c *gin.Context) {
 	// - include Attributes, VersionedAttributes and ServerComponentyType relations
 	mods := []qm.QueryMod{
 		qm.Load("Attributes"),
-		qm.Load("VersionedAttributes",
-			qm.Where(
-				`(namespace, created_at) IN (SELECT namespace, MAX(created_at) FROM versioned_attributes WHERE server_id=? GROUP BY namespace, server_component_id)`,
-				srv.ID,
-			),
-		),
+		qm.Load("VersionedAttributes"),
```

Comment:
We don't want to always return every set of versioned attributes only the most recent ones. I'm guessing you are changing this since you are calling an endpoint that isn't a server endpoint now so it failed? If so then I have some ideas on how we can fix it.

Reply:
Yep, I was trying to tackle this in the next set of changes I have ready. I've pushed a change -  [8e55dc8](https://github.com/metal-toolbox/hollow-serverservice/pull/101/commits/8e55dc823ad77ee5c3246adc52751c84a43f09f4) to limit the VersionedAttributes result set to the latest.